code/mob/living/critter/ai/flock/flocktasks.dm

```diff
@@ -76,6 +76,12 @@ butcher
 	can_be_adjacent_to_target = FALSE
 	max_dist = 0
 
+	New()
+		..()
+		var/datum/aiTask/succeedable/move/movesubtask = subtasks[subtask_index]
+		if(istype(movesubtask))
+			movesubtask.max_path_dist = 300
```

Comment:
Is there some way to avoid copy+pasting this?

Reply:
I don't think so, the reason it's copy pasted is because the targetable tasks are horrible hacks that don't actually share inheritance. I guess you could define a proc to set up an AItask as targetable and then call that from the new of all the targetables, but that might just be even more cursed.

Reply:
I guess I could add a flag to the task for long range moves or not, but tbh I think that might be better to do in another pr where I also slightly refractor timed tasks to use it too.